src/public/scss/changelist.scss

```diff
@@ -6,8 +6,16 @@
       width: 1rem;
     }
   }
-
+  
   &__count {
     color: $gray-600;
   }
+
+  &__list-actions {
+    display: flex;
+  }
+}
+
+.btn {
+  margin-left: 5px;
```

Comment:
You're adding this margin on *all* buttons on this page. You can move this into the `&__list-actions` block to scope it to buttons inside the `admin-changelist__list-actions` div